terra/contracts/wormhole/src/contract.rs

```diff
@@ -261,6 +261,13 @@ fn vaa_update_guardian_set(deps: DepsMut, env: Env, data: &Vec<u8>) -> StdResult
         return ContractError::GuardianSetIndexIncreaseError.std_err();
     }
 
+    /* A new GuardianSet with zero addresses should never be accepted,
+     * it would break assumptions of the underlying contracts
+     */
+    if new_guardian_set.addresses.len() == 0
```

Comment:
style nit: `if new_guardian_set.addresses.is_empty()`

If you run `cargo clippy` it will catch stuff like this though you might want to apply #983 first to fix unrelated warnings.

Reply:
I can rebase once we land #983, but I'll also fix this suggestion now.

I like your suggestion on `cargo clippy`, let's just make it part of the build to keep us honest going forward.

Reply:
This line as also missing a curly brace (`{`) so it won't actually build but maybe that's intentional.